Makefile

```diff
@@ -53,3 +53,25 @@ image/build/push: image/build image/push
 test/unit:
 	@echo Running tests:
 	go test -v -race -cover ./pkg/...
+
+.PHONY: minikube/prepare/local
+minikube/prepare/local:
+	-kubectl create namespace ${NAMESPACE}
+	kubectl apply -f deploy/crds
+	kubectl apply -f deploy/roles -n ${NAMESPACE}
+	kubectl apply -f deploy/cluster_roles
+	kubectl apply -f deploy/examples/Grafana.yaml -n ${NAMESPACE}
+
+
+.PHONY: minikube/deploy
+minikube/deploy: minikube/prepare/local
+	kubectl apply -f deploy/operator.yaml -n ${NAMESPACE}
+
+.PHONY: minikube/stop
```

Comment:
Going to rename this to something less ambiguous to prevent confusion between the recipe and minikube command